### PR TITLE
glide 0.8.0

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

File: Library/Formula/glide.rb [hunk 1]
```diff
@@ -41,17 +41,17 @@ class Glide < Formula
...
 
   def install
-    (buildpath + "src/github.com/Masterminds/glide").install "glide.go", "cmd", "gb", "util", "yaml"
+    (buildpath + "src/github.com/Masterminds/glide").install "glide.go", "cfg", "cmd", "dependency", "gb", "msg", "util"
 
     ENV["GOPATH"] = buildpath
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "-o", "glide", "-ldflags", "-X main.version 0.7.2", "#{buildpath}/src/github.com/Masterminds/glide/glide.go"
+    system "go", "build", "-o", "glide", "-ldflags", "-X main.version 0.8.0", "#{buildpath}/src/github.com/Masterminds/glide/glide.go"
```
Comment (mikemcquaid) [inline Library/Formula/glide.rb]: Maybe just embed `#{version}` here?

File: Library/Formula/glide.rb [hunk 2]
```diff
@@ -41,17 +41,17 @@ class Glide < Formula
...
     bin.install "glide"
   end
 
   test do
     version = pipe_output("#{bin}/glide --version")
-    assert_match /0.7.2/, version
+    assert_match /0.8.0/, version
```
Comment (mikemcquaid) [inline Library/Formula/glide.rb]: Maybe just embed `#{version}` here?

**Comment by mattfarina** [comment]
Updated per the feedback from @mikemcquaid

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**